action.yml

```
@@ -396,18 +430,38 @@ runs:
               'auto', gha_target_python_version
           }
       ):
-          command.extend([
-              '--controller',
-              f"docker:default,python={gha_controller_python_version}",
-              '--target',
+          controller_opts = (
+              "docker:default,"
+              f"python={gha_controller_python_version}"
+          )
+          if gha_controller_docker_opts:
+              controller_opts += f",{gha_controller_docker_opts}"
+          command.extend(['--controller', controller_opts])
```

How about keeping it as a list?
```suggestion
          controller_opts = [
              "docker:default",
              f"python={gha_controller_python_version}",
          ]
          if gha_controller_docker_opts:
              controller_opts.append(gha_controller_docker_opts)
          command.extend(['--controller', ','.join(controller_opts)])
```
(Same below.)